Sources/FINNFilterImplementation/FilterBuilder/RangeFilterInfoBuilder.swift

```diff
@@ -219,4 +221,76 @@ private extension RangeFilterInfoBuilder {
             appearanceProperties: appearanceProperties
         )
     }
+
+    func buildRangeFilterInfoForMCMarket(from filterData: FilterData) -> RangeFilterInfoType? {
+        let parameterName = filterData.parameterName
+        let name = filterData.title
+        let lowValue: Int
+        let highValue: Int
+        let increment: Int
+        let unit: String
+        let rangeBoundsOffsets: RangeFilterInfo.RangeBoundsOffsets
+        let accessibilityValues: RangeFilterInfo.AccessibilityValues
+        let appearanceProperties: RangeFilterInfo.AppearenceProperties
+
+        guard let filterKey = FilterKey(stringValue: filterData.parameterName) else {
+            return nil
+        }
+        switch filterKey {
+        case .year:
+            lowValue = 1950
+            highValue = Calendar.current.component(.year, from: Date())
+            unit = "år"
+            rangeBoundsOffsets = (hasLowerBoundOffset: true, hasUpperBoundOffset: true)
+            increment = 1
+            accessibilityValues = (stepIncrement: nil, valueSuffix: nil)
+            appearanceProperties = (usesSmallNumberInputFont: false, displaysUnitInNumberInput: false, isCurrencyValueRange: false)
+        case .engineEffect:
+            lowValue = 0
+            highValue = 200
+            unit = "hk"
+            rangeBoundsOffsets = (hasLowerBoundOffset: false, hasUpperBoundOffset: true)
+            increment = 10
```

Comment:
I think we can start using arrays of values instead of increment. Not sure what they should be, but that's the goal I think.

Reply:
I agree 👍